op-chain-ops/deployer/pipeline/opchain.go

```diff
@@ -91,13 +94,46 @@ func DeployOPChain(ctx context.Context, env *Env, artifactsFS foundry.StatDirFs,
 		DelayedWETHPermissionedGameProxyAddress:   dco.DelayedWETHPermissionedGameProxy,
 		DelayedWETHPermissionlessGameProxyAddress: dco.DelayedWETHPermissionlessGameProxy,
 	})
+
+	currentBlock, _ := env.L1Client.BlockNumber(ctx)
+	block, _ := env.L1Client.BlockByNumber(ctx, big.NewInt(int64(currentBlock)))
+	currentBlockHash := block.Hash()
+
+	// If any of the implementations addresses (excluding OpcmProxy) are empty,
+	// we need to set them using the addresses of the corresponding proxies.
+	// The reason these might be empty is because we're only invoking DeployOPChain.s.sol as part of the pipeline.
+	// TODO: Need to initialize 'mipsSingletonAddress' and 'preimageOracleSingletonAddress'
```

Comment:
Should we track this in https://github.com/ethereum-optimism/platforms-team/issues/284 so we don't forget to do this?

Reply:
I tracked this - on mobile right now. But it's in the original tracking link.